geocity/apps/submissions/templates/submissions/_submission_actions.html

```diff
@@ -94,15 +94,27 @@ <h4>{% translate "Informations générales" %}</h4>
                     {% for field in forms.amend.get_base_fields %}
                         {% bootstrap_field field layout='horizontal'%}
                     {% endfor %}
-
+                    <div class="container">
                     {% for object_type, fields in forms.amend.get_fields_by_form %}
                         {% if fields %}
-                            <h4 class="my-5">{{ object_type }}</h4>
-                            {% for field in fields %}
+                        <h4 class="my-5">{{ object_type }}</h4>
+                          <div class="row">
+                            {% for field, amend_field_is_visible_by_author in fields %}
+                              <div class="col-1" style="padding-top:7px;">
+                                {% if amend_field_is_visible_by_author %}
+                                  <i class="fa fa-eye" style="color:rgb(140, 199, 30);" aria-hidden="true" title="{% trans 'Visible par le requérant' %}"></i>
```

Comment:
put color in hex

Reply:
Do the same for mains.css 715 to 723.

How to do it ? 
- Hover the color
- Click on the upper bar, it changes the format